README.md

```diff
@@ -1,2 +1,23 @@
 # j2objc-common-libs-e2e-test
 End-to-end tests for translating, compiling, and linking common Java libraries into Objective C
+using [J2ObjC](http://j2objc.org) and the [J2ObjC Gradle Plugin](https://github.com/j2objc-gradle).
+
+This is a subproject of [j2objc-gradle](https://github.com/j2objc-gradle)
+and [maintained by some of the same people](NOTICE). Like j2objc-gradle, this project is not affiliated
+with Google.
+
+__Goals:__
```

Comment:
Use ** instead of __. We already have that as a convention in CONTRIBUTING.md in the j2objc-gradle repo.